website/docs/r/servicebus_subscription.html.markdown

```diff
@@ -73,6 +73,18 @@ The following arguments are supported:
 
 * `status` - (Optional) The status of the Subscription. Possible values are `Active`,`ReceiveDisabled`, or `Disabled`. Defaults to `Active`.
 
+* `client_scoped_subscription_enabled` - (Optional)  whether the subscription has an affinity to the client id. Defaults to `False`. 
+
+~> **NOTE:** Client Scoped Subscription can only be used for JMS subscription.
```

Comment:
what is a JMS subscription?

Reply:
Thanks Kt for the review, as the link that I put in the pr description, the full name of JMS subscription is Java message service (JMS) subscriptions: _While these are semantically similar to the [Subscriptions](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-queues-topics-subscriptions#topics-and-subscriptions) (that is, exist on a topic and enable publish/subscribe semantics), the Java Message Service spec introduces the concepts of Shared, Unshared, **Durable, and Non-durable attributes for a given subscription._